src/site/content/en/blog/async-clipboard/index.md

```diff
@@ -209,6 +209,23 @@ async function getClipboardContents() {
 }
 ```
 
+### Working with pasted files
+
+It is useful for users to be able to use clipboard keyboard shortcuts such as
+<kbd>ctrl</kbd>+<kbd>c</kbd> and <kbd>ctrl</kbd>+<kbd>v</kbd>.
+Chromium exposes _read-only_ files on the clipboard as outlined below.
```

Comment:
I don't think the code example illustrates the read-only nature of clipboard items. Specifically, if such items were read/write, this sample would still work.

Reply:
I have added a comment that clarifies this.